src/client/utils/hooks/useFeature.ts

```diff
@@ -12,24 +14,29 @@ type Env = 'staging' | 'production'
 interface FeatureConfig {
   name: Features
   envs: Env[]
-  markets: Market[]
+  markets: MarketLabel[]
 }
 
 const Config: readonly FeatureConfig[] = [
   {
     name: Features.OFFER_PAGE_INSURANCE_TOGGLE,
     envs: ['staging', 'production'],
-    markets: [Market.Se],
+    markets: ['SE'],
   },
   {
     name: Features.CHECKOUT_CREDIT_CHECK,
     envs: ['staging', 'production'],
-    markets: [Market.No],
+    markets: ['NO'],
   },
   {
     name: Features.TEST_FEATURE,
     envs: ['staging'],
-    markets: [Market.Se],
+    markets: ['SE'],
+  },
+  {
+    name: Features.QUOTE_CART_API,
+    envs: process.env.ENABLE_QUOTE_CART_API === 'true' ? ['staging'] : [],
```

Comment:
I'm trying this out to be able to toggle features based on an environment variable instead of requiring code changes -- let me know what you think!

Reply:
As we've discussed, maybe it would be good to move all config to the .env vars
Here is a ticket I created https://hedvig.atlassian.net/browse/GRW-599